### PR TITLE
Deprecate NexposeCreateIncidentsFromAssets

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/36753

## Description
deprecating the script as it may cause performance issues.

### CODE REVIEW

File: Packs/Rapid7_Nexpose/ReleaseNotes/1_1_1.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### NexposeCreateIncidentsFromAssets
+- Deprecated.
```
Comment (Itay4) [inline Packs/Rapid7_Nexpose/ReleaseNotes/1_1_1.md:4]: need to mention there is no replacement

File: Packs/Rapid7_Nexpose/Scripts/script-NexposeCreateIncidentsFromAssets.yml [hunk 2]
```diff
@@ -83,6 +84,7 @@ subtype: python2
 tags:
 - nexpose
 comment: |-
+  Deprecated.
```
Comment (Itay4) [inline Packs/Rapid7_Nexpose/Scripts/script-NexposeCreateIncidentsFromAssets.yml:87]: need to mention there is no replacement

**Review by Itay4 — APPROVED** [review]

File: Packs/Rapid7_Nexpose/Scripts/script-NexposeCreateIncidentsFromAssets.yml (hunk 2 above)
Comment (yaakovi) [reply Packs/Rapid7_Nexpose/Scripts/script-NexposeCreateIncidentsFromAssets.yml:87]:
```suggestion
  Deprecated. No available replacement.
```

File: Packs/Rapid7_Nexpose/ReleaseNotes/1_1_1.md (hunk 1 above)
Comment (yaakovi) [reply Packs/Rapid7_Nexpose/ReleaseNotes/1_1_1.md:4]:
```suggestion
- Deprecated. No available replacement.
```

**Review by David-BMS — COMMENTED** [review]
Approved

**Pull request merged**